controllers/auth.mjs

```diff
@@ -1,17 +1,39 @@
-import { Router } from 'express';
+//import { Router } from 'express';
+import express from 'express';
+import bcrypt from 'bcryptjs';
+import jwt from 'jsonwebtoken'
+
 import db from '../database';
 
-const router = Router();
+const router = express.Router();
+//const router = Router();
 
 router.post('/signup', async (req, res) => {
   const { email, password } = req.body;
-  const user = await db('Accounts').where({ email });
+  const [user] = await db('Accounts').where({ email });
   if (!user) {
-    await db('Accounts').insert({ email, password, account_created: Date.now() });
-    res.redirect('/signin');
+    const hash = bcrypt.hashSync(password, 8);
+    await db('Accounts').insert({ email, password: hash, account_created: Date.now() });
```

Comment:
You should put try-catch here and response with Internal Server Error (500) if the server couldn't access the database. Waiting for connection to timeout feels very wrong =).

Reply:
How doing it?

Reply:
Nevermind, that external error handler solves that.